wcomponents-core/src/main/java/com/github/bordertech/wcomponents/render/webxml/WTextAreaRenderer.java

```diff
@@ -60,7 +58,10 @@ public void doRender(final WComponent component, final WebXmlRenderContext rende
 			xml.append("<ui:rtf />");
 		}
 
-		xml.appendEscaped(textArea.getText());
+		String textString = textArea.getText();

```

Comment:
Just to make sure I understand....

The HTML sanitizing will make it safe to include in XML?

Why we are not escaping the text for "ReadOnly" and "Rich Text" field? How is it different to "not readonly"?

Is it dangerous to not escape and allow "malicious" script through?